### DISCUSSION TITLE
Put in HTML data via GraphQL - page mutation

### DISCUSSION THREAD

**Discussion by rhvitfeldt** [requarks/wiki#5947] [Help / Questions] [answered]
Hi!

We'd like to, via GraphQL, create and populate a page with HTML code. We have come so far that we can make requests successfully, create a page with just a single word for example. But we'd like to create a page with HTML code that we add via the GraphQL request. Problem is, with HTML and all the special characters, we get this error:

{"errors":[{"message":"Syntax Error: Cannot parse the unexpected character \"/\".","locations":[{"line":4,"column":67}],"extensions":{"code":"GRAPHQL_PARSE_FAILED"}}]}

How can we put in content with GraphQL with just a lot of data?

**Comment by NGPixel** [comment, marked as answer]
That's because you didn't properly escape your strings.

**Reply by rhvitfeldt** [reply]
> That's because you didn't properly escape your strings.

Okay. How should it be formatted? This is an example when we get the error.
```

mutation Page {
  pages {
    create (
        content : "<p>Welcome to xxx!</p><p><a href="/home/xx/">xx</a></p><p><a href="/home/xx/">xx</a></p>",
        description: "Test",
        editor: "code",
        locale: "en",
        path: "/home/xx/xx/xx/110099",
        tags: ["test"],
        title: "110099 - xx",
        isPublished: true,
        isPrivate: false
        ) {
          responseResult {
          succeeded,
          errorCode,
          slug,
          message
        },
        page {
          id,
          path,
          title
        }
      }
  }
}
```

**Reply by NGPixel** [reply]
Like I said, you didn't escape your string. You need to escape double quotes in your content value. I suggest reading tutorials on how to properly escape values in programming languages.

**Reply by rhvitfeldt** [reply]
> Like I said, you didn't escape your string. You need to escape double quotes in your content value. I suggest reading tutorials on how to properly escape values in programming languages.

Ok. Thank you.

Appreciate your work and help.
